README.md

```diff
@@ -436,10 +436,12 @@ tests (containing tests for backend specific components, including tests for the
 different types of endpoints) and Database Backend tests (like the ICAT backend tests,
 but covering only the most used aspects of the API).
 
-The configuration file (`config.json`) contains two options that will be used during the
+The configuration file (`config.yaml`) contains two options that will be used during the
 testing of the API. Set `test_user_credentials` and `test_mechanism` appropriately for
-your test environment, using `config.json.example` as a reference. The tests require a
-connection to an instance of ICAT 5, so set the rest of the config as needed. These tests have been written to assume ICAT 5 has been installed so please update to that or test with an older version of the API.
+
```

Comment:
This blank line looks to be causing some formatting issues (https://github.com/ral-facilities/datagateway-api/blob/1e08ed90d1243bd335590a42e93ef99c34a57e05/README.md#running-tests). Could you resolve that please (by removing the blank line)?

Reply:
Fixed in latest commit